lib/src/flutter/runners/gherkin_integration_test_runner.dart

```diff
@@ -469,6 +484,23 @@ abstract class GherkinIntegrationTestRunner {
   ) {
     return tagExpression == null || tagExpression.isEmpty
         ? true
-        : _tagExpressionEvaluator.evaluate(tagExpression, tags!.toList());
+        : _tagExpressionEvaluator.evaluate(
+            tagExpression,
+            tags!.toList(growable: false),
+          );
+  }
+
+  bool _isNegativeResult(StepExecutionResult result) {
+    return result == StepExecutionResult.error ||
+        result == StepExecutionResult.fail ||
+        result == StepExecutionResult.timeout;
+  }
+
+  Future<void> _pumpAndSettle(WidgetTester tester) async {
+    await tester.pumpAndSettle(
+      const Duration(milliseconds: 200),
+      EnginePhase.sendSemanticsUpdate,
+      const Duration(milliseconds: 2000),
```

Comment:
@jonsamwell 
I think 2 seconds are far too few in comparison to default 10 minutes! Our tests fail now as  the app displays a splash screen which has some random latency. Do you mind changing it to 30 seconds or even 2 minutes?

Reply:
Yes, I hadn't considered this.  I think it might be best to allow this to be specified, the same with the finalization pump as well.

Reply:
@golane-august appreciate your help review!

Reply:
@golane-august see the release note for 3.0,0-rc-11 as I've added an option to handle the app lifecycle pump yourself.